src/smash-league.js

```diff
@@ -266,12 +271,16 @@ const commitInProgress = rankingObj => {
     return result
 }
 
-const getPlayersThatCanBeChallenged = (playerPlace, playerRange, rankingTable) => {
+const getPlayersThatCanBeChallenged = (playerPlace, playerRange, completeRanking, playerId) => {
+    const { in_progress, ranking } = completeRanking
+    console.log(`culos ${in_progress} y: `, ranking)
```

Comment:
Lulz... please delete that log message. 😅

Reply:
lol, right, I forgot... 🤣